thirdparty/openjdk/zimbra-openjdk/debian/rules

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/make -f
 export DEB_BUILD_OPTIONS=nocheck
+export CFLAGS="-w"
```

Comment:
Could you please write a comment describing why we need this.

Reply:
I can understand that we are ignoring warnings here.

Reply:
to supres following warnings used the "-w" flag. 
`/root/ubuntutest/packages/thirdparty/openjdk/build/UBUNTU18_64/zimbra-openjdk/hotspot/src/share/vm/adlc/output_c.cpp:546:19: err
or: ‘%*d’ directive output between 1 and 2147483647 bytes may cause result to exceed ‘INT_MAX’ [-Werror=format-overflow=]
     sprintf(args, "0x%0*x, 0x%0*x, %*d",
                   ^~~~~~~~~~~~~~~~~~~~~`